sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -79,76 +79,84 @@ class Std {
     }
   }
 
-  private object Codepoint extends Val.Builtin1("str") {
+  private object Codepoint extends Val.Builtin1("codepoint", "str") {
     def evalRhs(str: Val, ev: EvalScope, pos: Position): Val =
       Val.Num(pos, str.asString.charAt(0).toInt)
   }
 
-  private object ObjectHas extends Val.Builtin2("o", "f") {
+  private object ObjectHas extends Val.Builtin2("objectHas", "o", "f") {
     def evalRhs(o: Val, f: Val, ev: EvalScope, pos: Position): Val =
       Val.bool(pos, o.asObj.containsVisibleKey(f.asString))
     override def specialize(args: Array[Expr]) = args match {
       case Array(o, s: Val.Str) => (new SpecF(s.value), Array(o))
       case _ => null
     }
-    private class SpecF(f: String) extends Val.Builtin1("o") {
+    private class SpecF(f: String) extends Val.Builtin1("objectHas", "o") {
       def evalRhs(o: Val, ev: EvalScope, pos: Position): Val =
         Val.bool(pos, o.asObj.containsVisibleKey(f))
     }
   }
 
-  private object ObjectHasAll extends Val.Builtin2("o", "f") {
+  private object ObjectHasAll extends Val.Builtin2("objectHasAll", "o", "f") {
     def evalRhs(o: Val, f: Val, ev: EvalScope, pos: Position): Val =
       Val.bool(pos, o.asObj.containsKey(f.asString))
     override def specialize(args: Array[Expr]) = args match {
       case Array(o, s: Val.Str) => (new SpecF(s.value), Array(o))
       case _ => null
     }
-    class SpecF(f: String) extends Val.Builtin1("o") {
+    class SpecF(f: String) extends Val.Builtin1("objectHasAll", "o") {
       def evalRhs(o: Val, ev: EvalScope, pos: Position): Val =
         Val.bool(pos, o.asObj.containsKey(f))
     }
   }
 
-  private object ObjectFields extends Val.Builtin1("o") {
+  private object ObjectFields extends Val.Builtin1("objectHasAll", "o") {
```

Comment:
Typo: this should be 

```suggestion
  private object ObjectFields extends Val.Builtin1("objectFields", "o") {
```